example/lib/ui/device_list.dart

```diff
@@ -0,0 +1,149 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_reactive_ble/flutter_reactive_ble.dart';
+
+import '../src/utils.dart';
+import '../src/widgets.dart';
+import 'device_detail_screen.dart';
+
+final _uuid = Uuid.parse('0000FE0F-0000-1000-8000-00805F9B34FB');
```

Comment:
I think we should let the user fil in the uuid since this will be only related to lights and this is a general purpose library,

Reply:
Should this be the default value when input is empty?

Reply:
No better give an error since we really require to filter on an uuid